client/components/web-preview/index.jsx

```diff
@@ -47,8 +56,13 @@ const WebPreview = React.createClass( {
 	getDefaultProps() {
 		return {
 			showExternal: true,
```

Comment:
Yeah :) Also once you save customizations the button could appear instantly. Though at that point we'd also have the "Saved!" notice to link to site.